MantidQt/CustomInterfaces/src/Indirect/ISISDiagnostics.cpp

```diff
@@ -467,5 +467,23 @@ void ISISDiagnostics::pbRunFinished() {
   m_uiForm.dsInputFiles->setEnabled(true);
 }
 
+/**
+ * Handles mantid plotting
+ */
+void ISISDiagnostics::plotClicked() {
+  bool plot = checkADSForPlotSaveWorkspace(m_pythonExportWsName, true);
+  if (plot)
```

Comment:
This could be turned into
`if (checkADSForPlotSaveWorkspace(m_pythonExportWsName, true);`